### PR TITLE
feat: add tmuxinator completion spec.

### PR DESCRIPTION
**Pull request by kmjohny**
Add new completion spec for [tmuxinator](https://github.com/tmuxinator/tmuxinator)

### CODE REVIEW

File: src/tmuxinator.ts [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+      ],
+    },
+    {
+      name: "debug",
+      description: "Output the shell commands that are generated by tmuxinator",
+      options: [
+        {
+          name: ["-a", "--attach"],
+          description: "Attach to tmux session after creation",
+          args: {},
```
Comment (SeparateRecords) [inline src/tmuxinator.ts:85]: Would you be able to give this a name? Description not necessary, name just helps in the popup

File: src/tmuxinator.ts [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+      name: "doctor",
+      description: "Look for problems in your configuration",
+    },
+    {
+      name: "help",
+      description: "Describe available commands or one specific command",
+      args: {
+        name: "command",
+        description: "The tmuxinator commands",
+        generators: commands,
```
Comment (SeparateRecords) [inline src/tmuxinator.ts:126]: I think you might be able to just use `template: "help"` instead? It displays all the commands on the parent spec object as suggestions. Unless there's some other behavior (eg. user-defined commands) that should be fine.

**Review by SeparateRecords — COMMENTED** [review]
Looks great! As is, we could definitely merge it, looks like everything would work fine. Just 2 small notes then I think it'll be perfect.

File: src/tmuxinator.ts (hunk 1 above)
Comment (kmjohny) [reply src/tmuxinator.ts]: Done!

File: src/tmuxinator.ts (hunk 2 above)
Comment (kmjohny) [reply src/tmuxinator.ts]: Makes sense. Done!.

**Review by SeparateRecords — APPROVED** [review]
Fantastic! Thank you so much for this. Looks brilliant.

**Pull request merged**